### PR TITLE
[Conf] Add nfs suites for Reef

### PR DESCRIPTION
**Pull request by pranavprakash20**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by MohitBis — DISMISSED** [review]
LGTM

File: conf/reef/nfs/tier-0.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+globals:
```
Comment (vamahaja) [inline conf/reef/nfs/tier-0.yaml:1]: Change suite name to `tier0`

File: conf/reef/nfs/tier-1-8node.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
+globals:
```
Comment (vamahaja) [inline conf/reef/nfs/tier-1-8node.yaml:1]: Change name to `12node-1admin-4client`

File: suites/reef/nfs/tier-0_nfs_ganesha.yaml [hunk 3]
```diff
@@ -0,0 +1,260 @@
+#===============================================================================================
```
Comment (vamahaja) [inline suites/reef/nfs/tier-0_nfs_ganesha.yaml:1]: Change `tier-0` to `tier0` and replace `_` with `-`

File: suites/reef/nfs/tier-1_nfs_ganesha.yaml [hunk 4]
```diff
@@ -0,0 +1,183 @@
+#===============================================================================================
```
Comment (vamahaja) [inline suites/reef/nfs/tier-1_nfs_ganesha.yaml:1]: Same as above

**Review by vamahaja — CHANGES_REQUESTED** [review]

**Comment by pranavprakash20** [comment]
changes made

**Review by vamahaja — APPROVED** [review]

**Pull request merged**